tests/framework/common.go

```diff
@@ -1007,11 +995,50 @@ func (td *OsmTestData) AddNsToMesh(shouldInjectSidecar bool, ns ...string) error
 	return nil
 }
 
-// UpdateOSMConfig updates OSM configmap
-func (td *OsmTestData) UpdateOSMConfig(key, value string) error {
-	patch := []byte(fmt.Sprintf(`{"data": {%q: %q}}`, key, value))
-	_, err := td.Client.CoreV1().ConfigMaps(td.OsmNamespace).Patch(context.TODO(), "osm-config", types.StrategicMergePatchType, patch, metav1.PatchOptions{})
-	return err
+// CreateDefaultOSMConfig creates a MeshConfig object with default settings
+func (td *OsmTestData) CreateDefaultOSMConfig() *v1alpha1.MeshConfig {
```

Comment:
Why is this required? Who is calling it?

Reply:
Forgot to remove.

This method was added when initially in the test cases, meshConfig object is created by `CreateDefaultOSMConfig`, rather than fetched from client.